scripts/convert-requirements-to-conda-yml.py

```diff
@@ -55,38 +45,27 @@ def build_environment_from_requirements(cli_args):
     args = parser.parse_args(cli_args)
     requirements_files = args.req
 
-    pip_requirements = set([])
+    pip_requirements = set()
     # conda likes it when you list pip if you're using pip.
     conda_requirements = set(['pip'])
     for requirement_file in requirements_files:
-        for line in open(requirement_file):
-            line = line.strip()
-
-            # Blank line or comment
-            if len(line) == 0 or line.startswith('#'):
-                continue
-
-            # Checked out from scm
-            if line.startswith(tuple(SCM_MAP.keys())):
-                pip_requirements.add(line)
-                conda_requirements.add(SCM_MAP[line.split('+')[0]])
-                continue
-
-            requirement = pkg_resources.Requirement.parse(line)
-            conda_forge_url = FEEDSTOCK_URL.format(
-                package=requirement.project_name.lower())
-            if (requests.get(conda_forge_url).status_code == 200 and not
-                    line.endswith('# pip-only')):
-                conda_requirements.add(line)
-            else:
-                pip_requirements.add(line)
-
-    conda_deps_string = '\n'.join(['- %s' % dep for dep in
-                                   sorted(conda_requirements,
-                                          key=lambda x: x.lower())])
-    pip_deps_string = '- pip:\n' + '\n'.join(['  - %s' % dep for dep in
-                                              sorted(pip_requirements,
-                                                     key=lambda x: x.lower())])
+        with open(requirement_file) as file:
+            for line in file:
+                line = line.strip()
+
+                # Blank line or comment
+                if len(line) == 0 or line.startswith('#'):
+                    continue
+
+                if line.endswith('# pip-only'):
+                    pip_requirements.add(line)
+                else:
+                    conda_requirements.add(line)
```

Comment:
Just to confirm, so if we wanted to include a package here in `requirements.txt` that's installed from a git or mercurial source tree, we'd also need to include the `# pip-only` suffix, right?

Reply:
Yes I think so... anything that we want to `pip install` should be marked `# pip-only` so that it goes in the `pip` subsection of the environment yml.

Reply:
Works for me!  And I did test this with a sample `git+<repo>@<rev>#egg=<package>` install in a requirements.txt.  Works fine with a trailing `# pip-only` as we'd expect, even with the `#egg` in there.